README.md

```diff
@@ -1,14 +1,88 @@
-# Project
+# Azure AI Evaluation GitHub Action
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+This GitHub Action enables offline evaluation of AI models within your CI/CD pipelines. It is designed to streamline the evaluation process, allowing you to assess model performance and make informed decisions before deploying to production. 
 
-As the maintainer of this project, please make a few updates:
+Offline evaluation involves testing AI models using test datasets to measure their performance on various quality and safety metrics such as fluency, coherence and content safety. After selecting a model in the [Azure AI Model Catalog](https://azure.microsoft.com/en-us/products/ai-model-catalog?msockid=1f44c87dd9fa6d1e257fdd6dd8406c42) or [GitHub Model marketplace](https://github.com/marketplace/models), offline pre-production evaluation is crucial for AI application validation during integration testing, allowing developers to identify potential issues and make improvements before deploying the model or application to production (e.g., when updating system meta prompts). 
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+## Features
+* **Automated Evaluation:** Integrate offline evaluation into your CI/CD workflows to automate the pre-production assessment of AI models. 
+* **Out of the Box and Custom Evaluators:** Leverage existing evaluators provided by the [Azure AI Evaluation SDK](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/develop/evaluate-sdk) or add your own custom evaluators. The following evaluators are supported: 
+  * BleuScoreEvaluator
+  * CoherenceEvaluator 
+  * ContentSafetyEvaluator 
+  * HateUnfairnessEvaluator 
+  * SelfHarmEvaluator 
+  * SexualEvaluator 
+  * ViolenceEvaluator 
+  * F1ScoreEvaluator 
+  * FluencyEvaluator 
+  * GleuScoreEvaluator 
+  * GroundednessEvaluator 
+  * MeteorScoreEvaluator 
+  * ProtectedMaterialEvaluator 
+  * QAEvaluator 
+  * RelevanceEvaluator 
+  * RetrievalEvaluator 
+  * RougeScoreEvaluator 
+  * SimilarityEvaluator 
+  * IndirectAttackEvaluator 
+* **Seamless Integration:** Easily integrate with existing GitHub workflows to run evaluation based on rules that you specify in your workflows (e.g., when changes are committed to feature flag configuration or system meta prompt files). 
```

Comment:
```suggestion
* **Seamless Integration:** Easily integrate with existing GitHub workflows to run evaluation based on rules that you specify in your workflows (e.g., when changes are committed to feature flag configuration or system prompt files). 
```

Reply:
Updated to make consistent with terminology elsewhere. Didn't catch this earlier.